lib/class-wp-rest-url-details-controller.php

```diff
@@ -164,12 +168,12 @@ public function permissions_check() {
 	 * Retrieves the document title from a remote URL.
 	 *
 	 * @param string $url The website url whose HTML we want to access.
-	 * @return array|WP_Error the HTTP response from the remote URL or error.
+	 * @return array|WP_Error the HTTP response from the remote URL, or an error.
```

Comment:
```suggestion
	 * @return string|WP_Error The HTTP response from the remote URL, or an error.
```

`wp_remote_retrieve_body` returns a `string`.